plugins/quickinspector/quickoverlay.cpp

```diff
@@ -50,6 +50,50 @@ extern Q_GUI_EXPORT QImage qt_gl_read_framebuffer(const QSize &size,
 QT_END_NAMESPACE
 
 namespace GammaRay {
+
+class QQuickItemPropertyCache {
+public:
+    static const QQuickItemPropertyCache &getPropertyCache(QQuickItem *item)
+    {
+        static QHash<QByteArray, QQuickItemPropertyCache> s_cache;
+        const auto meta = item->metaObject();
+        const auto className = QByteArray(meta->className());
```

Comment:
Connection this the object doesn't sound like the best idea.
Besides, the class name probably persists over the lifetime of the qobject?